src/js/main.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-env browser */
+var APP_ID = 'com.bitmovin.demo.webapp';
+var PLAYER_KEY = 'key';
+
+var player;
+var source = {
+  // AVC Stream
+  // dash : "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd",
+  // HEVC Stream
+  // dash : "https://bitmovin-a.akamaihd.net/content/multi-codec/hevc/stream.mpd"
+  // Widevine Stream
+  dash: 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd',
+  drm: {
+    widevine: {
+      LA_URL: 'https://widevine-proxy.appspot.com/proxy'
+    }
+  }
+};
+
+window.onload = function () {
+  setupControllerEvents();
+  setupPlayer();
+
+  var keySystem = webOSDev && webOSDev.DRM.Type.WIDEVINE;
+  var webosDrmAgent = getDrmAgent(keySystem);
+
+  // In the app is started shortly after webOS is rebooted the DRM system and CMD
+  // might not be fully ready to use for DRM playback. Therefore we should await
+  // drmAgents onsuccess callback before we try to load DRM source.
+  if (webosDrmAgent) {
+    isDrmLoaded(webosDrmAgent)
+      .then(function () {
+        return loadSource(source);
+      })
+      .catch(function (e) {
+        console.log('Error while loading drm Agent', e);
+      });
+    return;
+  }
+
+  // In case we don't have drmAgent available, just load the source normal way.
+  loadSource(source);
+};
+
+function setupPlayer () {
+  // add all necessary (and loaded) modules to the player core
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.polyfill.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player['engine-bitmovin'].default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player['container-mp4'].default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player['container-ts'].default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.mserenderer.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.abr.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.drm.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.xml.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.dash.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.hls.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.style.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.tizen.default);
```

Comment:
Are you sure we need `tizen` module in webos demo?

Also, I think you should include `webos` module.

Reply:
@mukulmishra18 Thx for spotting this, not sure how it got here. Any idea @jdmartins ?

Reply:
Changed to webos

Reply:
yes it should be changed